src/app/application.cpp

```diff
@@ -559,7 +557,6 @@ int Application::exec(const QStringList &params)
     }
 #endif // DISABLE_WEBUI
 #else
-    UIThemeManager::initInstance();
```

Comment:
Why did you move it above? I wouldn't initialize it too early unless there is strong reason.

Reply:
It is now placed where `iconprovider` is initialized previously.

Reply:
>It is now placed where iconprovider is initialized previously.

I see it myself. I asked you why do you initialize UIThemeManager here instead if its previous place.